kubetest2/kubetest2-gce/deployer/down.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package deployer
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"k8s.io/klog"
+	"k8s.io/test-infra/kubetest2/pkg/exec"
+)
+
+func (d *deployer) Down() error {
+	klog.Info("GCE deployer starting Down()")
+
+	if err := d.verifyFlags(); err != nil {
+		return fmt.Errorf("down could not verify flags: %s", err)
+	}
+
+	env := d.buildEnv()
```

Comment:
do we want to persist the project that we used in up somewhere?

have you tried running build/up/down separately?

Reply:
I lean towards not persisting the project. The user has to pass the project as a flag to --up anyway and I think avoiding producing artifacts when we don't have to is a good idea.

Running build/up/down separately has been tested and does work.